zhaquirks/__init__.py

```diff
@@ -66,18 +66,37 @@ class LocalDataCluster(CustomCluster):
 
     async def bind(self):
         """Prevent bind."""
+        self.log(
+            logging.DEBUG,
```

Comment:
I think just using `self.debug` is preferred, as that's what is used in zigpy all the time.

Reply:
I didn’t see the log mixin in the class hierarchy. I’ll look again.

Reply:
Ugh it is pulled in by CatchingTaskMixin

Reply:
Updated